docs/quickstart.md

```diff
@@ -66,10 +62,12 @@ $ make manager
 $ ./output/manager --create-crds-for-ftcs \
     --klog-logtostderr=false \
     --klog-log-file "./kubeadmiral.log" \
-    --kubeconfig "$HOME/.kube/kubeadmiral/kubeconfig.yaml" \
+    --kubeconfig "$HOME/.kube/kubeadmiral/kubeadmiral-host.yaml" \
```

Comment:
why do we have to repeat the word `kubeadmiral` in the path here?

Reply:
the generated kubeconfig files use the cluster names for convenience

Reply:
The names should include the cluster names, but is there a need to repeat `kubeadmiral`?

Reply:
are you suggesting naming the clusters without `kubeadmiral` in them? or not organizing the kubeconfigs in a `kubeadmiral` directory?

Reply:
The former. But given that the file name follows directly from the kind cluster name, let's leave it be for simplicity.